src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding_editor.yaml

```diff
@@ -0,0 +1,12 @@
+Type: Jupyter Notebook Extension
+Name: Codefolding in Editor
+Description: |
+  Enables the CodeMirror feature to allow codefolding in the Jupyter file
+  editor view.
+  Note that this also uses the codefolding hotkey from the codefolding
+  nbextension for the notebook view.
```

Comment:
Alternatively, i could add the configurator feature I made a while back to allow the parameters to go in a different config section to the enabled/disabled status...